src/components/Navbar.js

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+
+export const DesktopNavItems = styled.div`
+display:block;
+`;
+
+export const MobileQuery = styled(DesktopNavItems)`
+ @media screen and (max-width: 768px) {
+    display: none;
+  }
+`;
+
+export const Nav = styled.nav`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: rgb(0, 0, 0, 0.6);
+  color: #E3F7FC;
+  height: 5rem;
+  padding: 0 2rem;
+  z-index: 1;
+  width: 83%;
+`;
+
+export const StickyNav = styled(Nav)`
```

Comment:
I guess this is the answer to my question about Sticky before in this CR :) 
I will probably use it very soon myself. Overall clean code as always - nice to read and easy to follow.